examples/CMakeLists.txt

```
@@ -1269,3 +1269,19 @@ add_impactx_test(examples-scraping.py
     examples/scraping_beam/analysis_scraping.py
     OFF  # no plot script yet
 )
+
+# FODO cell with 2D space charge using envelope tracking ######################
+#
+# with space charge
+add_impactx_test(fodo-envelope-sc
+    examples/fodo_space_charge/input_fodo_envelope_sc.in
+    ON   # ImpactX MPI-parallel
+    examples/fodo_space_charge/analysis_fodo_envelope_sc.py
+    OFF  # no plot script yet
+)
+add_impactx_test(fodo-envelope-sc.py
```

I think that is the naming scheme we used for the FODO tests:
```suggestion
add_impactx_test(FODO.envelope.sc
    examples/fodo_space_charge/input_fodo_envelope_sc.in
    ON   # ImpactX MPI-parallel
    examples/fodo_space_charge/analysis_fodo_envelope_sc.py
    OFF  # no plot script yet
)
add_impactx_test(FODO.envelope.sc.py
```